dockerfiles/ci/buster/php-8.0/configure_shared_ext.sh

```diff
@@ -7,7 +7,6 @@ ${PHP_SRC_DIR}/configure \
     --disable-all \
     --enable-cgi \
     --enable-fpm \
-    --enable-pcntl \
```

Comment:
We should build pcntl as shared module here

Reply:
Just write `--enable-pcntl=shared` instead, like here: https://github.com/DataDog/dd-trace-php/blob/424a46b7a082d67fc9350e325257b24fe7a0098e/dockerfiles/ci/buster/php-8.0/configure_shared_ext.sh#L10 (or just remove it, so that it won't merge conflict)

Reply:
After rebasing on master, this is resolved.